libs/dh/metering-point/feature-upload-measurements/src/upload-page.ts

```diff
@@ -211,10 +198,9 @@ import { assertIsDefined } from '@energinet-datahub/dh/shared/util-assert';
     </watt-card>
   `,
 })
-export class DhMeasurementsUploadComponent {
+export class DhUploadMeasurementsComponent {
   private navigate = injectRelativeNavigate();
-  protected measurements = inject(DhMeasurementsUploadDataService);
-  private readonly csvParser: CsvParseService = inject(CsvParseService);
+  protected measurements = inject(DhUploadMeasurementsService);
```

Comment:
Nitpick: Why not private, protected is for inheritance?

Reply:
Good eye, I think it was protected because it used to be accessed from the template. But that is no longer the case, so private it is 🙂 

https://github.com/Energinet-DataHub/greenforce-frontend/pull/4485/commits/bab065b7003f8a9b32dc7d1aaeb3544c03790f08